lib/pipeline.js

```diff
@@ -511,9 +520,16 @@ class PipelineModel extends BaseModel {
 
             const userFactory = UserFactory.getInstance();
 
-            return userFactory.get({ username: Object.keys(this.admins)[0] })
+            return userFactory.get({
+                username: Object.keys(this.admins)[0],
+                scmContext: this.scmContext
+            })
```

Comment:
The indentation of this block looks funny. The closing brackets should line up with the `.then` on the next line.